frontend/components/WordmarkLogo.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, chakra, Icon, IconProps } from "@chakra-ui/react";
+import Link from "next/link";
+import React, { PropsWithChildren } from "react";
+
+interface WordmarkLogoLink {
+  url: string;
+}
+
+export const WordmarkLogoLink = chakra(
+  (props: PropsWithChildren<WordmarkLogoLink>) => {
+    return (
+      <Link href={props.url} passHref>
+        <Box as="a">{props.children}</Box>
+      </Link>
+    );
+  }
+);
+
+export const WordmarkLogo = chakra(
+  ({ width, height, color = "rgb(19,22,57)", ...otherProps }: IconProps) => {
+    color = color.toString(); //Color is of type Token<CSS.Property.Color, "colors">, so i have to transform it as a string
+    return (
+      <Icon width={width} height={height} viewBox="0 0 554 150" {...otherProps}>
```

Comment:
Not necessary to extract `width` and `height` as they can be kept within "otherProps"